### PR TITLE
[SyntaxBuilder] Make `String` conform to `ExpressibleAs` protocols

### PR DESCRIPTION
**Pull request by kimdv**
https://github.com/apple/swift-syntax/pull/335

### CODE REVIEW

**Comment by ahoppen** [comment]
https://github.com/apple/swift-syntax/pull/335

@swift-ci Please smoke test

**Comment by ahoppen** [comment]
apple/swift-syntax#335

@swift-ci Please smoke test

**Review by ahoppen — APPROVED** [review]

**Comment by ahoppen** [comment]
apple/swift-syntax#335

@swift-ci Please smoke test

File: utils/gyb_syntax_support/protocolsMap.py [hunk 1]
```diff
@@ -4,12 +4,20 @@
         'MemberDeclListItem',
         'SyntaxBuildable'
     ],
-    'StmtBuildable': [
```
Comment (kimdv) [inline utils/gyb_syntax_support/protocolsMap.py:7]: Made then in alphabetical order so it's easier to find them. 
Would you mind to verify that?

**Pull request merged**